smalltalksrc/VMMaker/CogOutOfLineLiteralsARMv8Compiler.class.st

```diff
@@ -90,11 +90,8 @@ CogOutOfLineLiteralsARMv8Compiler >> concretizeMoveMbrR [
 		ifTrue: [ :immediate9bitTwoComplementValue |
 			self
 				machineCodeAt: 0
-				put: (self movSize: 1 destinationRegister: destReg imm: 0 shift: 0).
-			self
-				machineCodeAt: 4
 				put: (self ldurbSize: 1 baseRegister: srcReg signedOffset: immediate9bitTwoComplementValue destinationRegister: destReg).
-			^ machineCodeSize := 8 ]
```

Comment:
Do we have a test for this one?
I remember the case, `ldurb` already zero extends so no need to zero it before.
Yet, I don't remember what caused it and why we needed to remove it. Was it a problem when `srcReg == destReg`?

Reply:
I don't remember the error, we can rollback and check. Anyway, there are failing tests (maybe) because this change, check: https://github.com/pharo-project/pharo-vm/pull/515